cypress/integration/ModelPage_spec.js

```diff
@@ -1,14 +1,12 @@
 describe('/state/:id', () => {
-  it(`Stay At Home has the correct title & description`, () => {
+  it(`State page has the correct title & description`, () => {
     cy.visit('/us/ny');
     cy.title().should(
       'eq',
-      'Real-time modeling and metrics to understand where we stand against COVID. - Covid Act Now',
-    );
-    cy.get('head meta[name="description"]').should(
-      'have.attr',
-      'content',
-      'Real-time modeling and metrics to understand where we stand against COVID. 50 states. 3,000+ counties. Click the map to dive in.',
+      'New York, NY - America’s COVID Warning System - Covid Act Now',
```

Comment:
FWIW- Normally "New York, NY" would refer to New York City.  Maybe this could be "New York (NY)"?